pipelines/protocolB.yaml

```diff
@@ -0,0 +1,278 @@
+parameters:
+  defects: false
+  linearize: false
+  crosstalk: false
+tasks:
+  # Bias generation:
+  cptBiasIsr:
+    class: lsst.cp.testing.CptIsrTask
+    config:
+      expectedExposureType: 'bias'
+      connections.ccdExposure: 'raw'
+      connections.outputExposure: 'cptBiasProc'
+      doWrite: true
+      doBias: false
+      doVariance: true
+      doLinearize: parameters.linearize
+      doDefect: parameters.defects
+      doCrosstalk: parameters.crosstalk
+      doBrighterFatter: false
+      doDark: false
+      doFlat: false
+      doApplyGains: false
+      doFringe: false
+  cptBiasCombine:
+    class: lsst.cp.pipe.cpCombine.CalibCombineTask
+    config:
+      connections.inputExpHandles: 'cptBiasProc'
+      connections.outputData: 'bias'
+      calibrationType: 'bias'
+      exposureScaling: "Unity"
+
+  # Dark generation:
+  cptDarkIsr:
+    class: lsst.cp.testing.CptIsrTask
+    config:
+      expectedExposureType: 'dark'
+      connections.ccdExposure: 'raw'
+      connections.bias: 'bias'
+      connections.outputExposure: 'cptDarkIsr'
+      doWrite: true
+      doBrighterFatter: false
+      doDark: false
+      doFlat: false
+      doApplyGains: false
+      doFringe: false
+      doLinearize: parameters.linearize
+      doDefect: parameters.defects
+  cptDark:
+    class: lsst.cp.pipe.cpDarkTask.CpDarkTask
+    config:
+      connections.inputExp: 'cptDarkIsr'
+      connections.outputExp: 'cptDarkProc'
+  cptDarkCombine:
+    class: lsst.cp.pipe.cpCombine.CalibCombineTask
+    config:
+      connections.inputExpHandles: 'cptDarkProc'
+      connections.outputData: 'dark'
+      calibrationType: 'dark'
+      exposureScaling: "DarkTime"
+      python: config.mask.append("CR")
+
+  # Flat generation:
+  cptFlatIsr:
+    class: lsst.cp.testing.CptIsrTask
+    config:
+      expectedExposureType: 'flat'
+      connections.ccdExposure: 'raw'
+      connections.bias: 'bias'
+      connections.dark: 'dark'
+      connections.outputExposure: 'cptFlatProc'
+      doWrite: true
+      doBrighterFatter: false
+      doFlat: false
+      doFringe: false
+      doApplyGains: false
+      doLinearize: parameters.linearize
+      doDefect: parameters.defects
+  cptFlatMeasure:
+    class: lsst.cp.pipe.cpFlatNormTask.CpFlatMeasureTask
+    config:
+      connections.inputExp: 'cptFlatProc'
+      connections.outputStats: 'cptFlatStats'
+  cptFlatNorm:
+    class: lsst.cp.pipe.cpFlatNormTask.CpFlatNormalizationTask
+    config:
+      connections.inputMDs: 'cptFlatStats'
+      connections.outputScales: 'cptFlatNormScales'
+  cptFlatCombine:
+    class: lsst.cp.pipe.cpCombine.CalibCombineByFilterTask
+    config:
+      connections.inputExpHandles: 'cptFlatProc'
+      connections.inputScales: 'cptFlatNormScales'
+      connections.outputData: 'flat'
+      calibrationType: 'flat'
+      exposureScaling: InputList
+      scalingLevel: AMP
+
+  #high-value (bright pixels) defects
+  cptBrightDefectIsr:
```

Comment:
I thought that we'd want this running on the combined calibs, not the raws?

Reply:
Good catch.  I'm not sure that was finished when I did the majority of the work on this ticket.  I'll fix this and the other defect step.